perun-db/oracle.sql

```diff
@@ -19,7 +19,9 @@ create table vos (
 	modified_by nvarchar2(1300) default user not null,
 	status char(1) default '0' not null,
 	created_by_uid integer,
-	modified_by_uid integer
+	modified_by_uid integer,
+	constraint VO_PK primary key (id),
+	constraint VO_U unique (name)
```

Comment:
So now some tables has constraints in definition of table itself, and same still have constraints below? This is very confusing.